docs/crio.8.md

```diff
@@ -114,6 +115,8 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--additional-devices**="": Devices to add to the containers  (default: [])
 
+**--allowed-annotations**="": list of experimental annotations that this runtime handler is allowed to process
```

Comment:
Is the comment accurate? I thought this PR was for the top level config and not specific to any runtime handler.

Reply:
We should also describe or document how this intersects with a runtime handler specific annotations.

Reply:
revised comment! Let me know what you think!